src/Parser/MarksmanshipHunter/Modules/Features/CooldownTracker.js

```diff
@@ -12,6 +14,19 @@ class CooldownTracker extends CoreCooldownTracker {
       ],
     },
   ];
+
+  castEventSpells = [
```

Comment:
Suggestion for future - if more spells are added here, perhaps it could be renamed as `ignoredSpells` or something similar as that's what's happening in `on_byPlayer_cast()`.